public/index.html

```diff
@@ -6,10 +6,10 @@
 
 	<title>Ochi</title>
 
-	<link rel='stylesheet' href='/global.css'>
-	<link rel='stylesheet' href='/build/bundle.css'>
+	<link rel='stylesheet' href='/src/global.css'>
+	<link rel='stylesheet' href='/src/build/bundle.css'>
 
-	<script defer src='/build/bundle.js'></script>
+	<script defer src='/src/build/bundle.js'></script>
```

Comment:
These paths no longer work. Did you want to move them as well?

Reply:
They should still be working :thinking:

Reply:
When I am running `npm run dev`, it is throwing a bunch of 404 errors. `src` directory referenced in above 4 lines no longer exists.

Reply:
ah, right, I was only running it through the backend! Should be fixed now.